<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TimeAttack</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>571</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Rallystage">
     <attribute name="title">
      <string>Rally stage</string>
     </attribute>
     <widget class="QPushButton" name="previousResult">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextResult">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>100</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="stageName">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>StageName</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="driverName">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DriverName</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="dnfBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>DNF</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>160</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>M:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="minutesBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>42</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>59</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>160</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>S:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="secondsBox">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>150</y>
        <width>42</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>59</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>160</y>
        <width>26</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ms:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="msecondsBox">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>150</y>
        <width>42</width>
        <height>31</height>
       </rect>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="finishedBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Finished</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Statistics">
     <attribute name="title">
      <string>Statistics</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QComboBox" name="rallyBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loadRally">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load rally</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rally name</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>564</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew_tournament">
    <property name="title">
     <string>Rally</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTracks">
    <property name="title">
     <string>Stages</string>
    </property>
    <addaction name="actionEdit"/>
   </widget>
   <widget class="QMenu" name="menuDrivers">
    <property name="title">
     <string>Drivers</string>
    </property>
    <addaction name="actionEdit_2"/>
   </widget>
   <addaction name="menuNew_tournament"/>
   <addaction name="menuTracks"/>
   <addaction name="menuDrivers"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionEdit_2">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
